script/test

```diff
@@ -49,15 +49,15 @@ const resourcePath = CONFIG.repositoryRootPath
 let executablePath
 if (process.platform === 'darwin') {
   const executablePaths = glob.sync(path.join(CONFIG.buildOutputPath, '*.app'))
-  assert(executablePaths.length === 1, `More than one application to run tests against was found. ${executablePaths.join(',')}`)
+  assert(executablePaths.length === 1, `A single application to run tests against was not found. ${executablePaths.join(',')}`)
```

Comment:
I personally think this is more confusing than before.

Are asserts horribly slow? This is a script for devs, not production code. It doesn't need to be very fast, just not extremely slow. I think two asserts lets us have helpful error messages.

Reply:
They are not slow. But when `executablePaths` gets printed they will know how many executables are found! There is no need to give a message for every situation.

Reply:
Okay. :shrug: